freqtrade/exchange/exchange.py

```diff
@@ -1595,6 +1595,36 @@ def get_historic_trades(self, pair: str,
             self._async_get_trade_history(pair=pair, since=since,
                                           until=until, from_id=from_id))
 
+    @retrier
+    def get_funding_fees_from_exchange(self, pair: str, since: Union[datetime, int]) -> float:
+        """
+            Returns the sum of all funding fees that were exchanged for a pair within a timeframe
+            :param pair: (e.g. ADA/USDT)
+            :param since: The earliest time of consideration for calculating funding fees,
+                in unix time or as a datetime
+        """
+
+        if not self.exchange_has("fetchFundingHistory"):
+            raise OperationalException(
+                f"fetch_funding_history() has not been implemented on ccxt.{self.name}")
+
+        if type(since) is datetime:
+            since = int(since.timestamp())
```

Comment:
This is wrong (either here, or below). ccxt expects timestamps to be in MS resolution - so you'll need to `*1000` values returned from `timestamp()`.